### ISSUE TITLE
Question on Action Normalization

### ISSUE THREAD

**Issue by AlexandreQ27** [cccedric/conrft#14]
First off, I want to express my sincere appreciation for the ConRFT framework! 

While diving into the code, I noticed that in the `ConsistencyPolicy_octo` class (in `serl_launcher/networks/actor_critic_nets.py`), the denoised action `x_0` is clipped to the `[-1, 1]` range using `x_0 = jnp.clip(x_0, -1, 1)`. 

I have a main question:  

For the Franka arm, where is the **denormalization** step implemented to convert the `[-1, 1]` clipped actions into physical control values (e.g., scaling to the robot’s actual joint ranges or Cartesian limits)? Is this handled implicitly in `FrankaEnv` or via other wrappers that I might have missed?  because 
i’m adapting the framework to a non-Franka robot arm. 

Thanks again for your work on this framework—any guidance would be greatly appreciated!

**Comment by Denghaoyuan123** [comment]
@AlexandreQ27 Hi, did you figure it out? I also have problem about this.

**Comment by AlexandreQ27** [comment]
> [@AlexandreQ27](https://github.com/AlexandreQ27) Hi, did you figure it out? I also have problem about this.

Hi, I've solved this issue.
The values from the SpaceMouse are just between -1 and 1.

**Comment by Denghaoyuan123** [comment]
> > [@AlexandreQ27](https://github.com/AlexandreQ27) Hi, did you figure it out? I also have problem about this.
> 
> Hi, I've solved this issue. The values from the SpaceMouse are just between -1 and 1.

Thanks! may I ask if your action space is delta-EEF (relative pose / velocity) control?
And what is the scaling factor applied to the SpaceMouse inputs (i.e., from -1, 1 to your actual control command)?

**Comment by AlexandreQ27** [comment]
> > > [@AlexandreQ27](https://github.com/AlexandreQ27) Hi, did you figure it out? I also have problem about this.
> > 
> > 
> > Hi, I've solved this issue. The values from the SpaceMouse are just between -1 and 1.
> 
> Thanks! may I ask if your action space is delta-EEF (relative pose / velocity) control? And what is the scaling factor applied to the SpaceMouse inputs (i.e., from -1, 1 to your actual control command)?

Yes, I use delta-EEF (relative pose/velocity) control. The scaling factor can be set based on your robot’s specs and control method – no fixed value.

Feel free to add my WeChat if you want to discuss more(ID:juven_alexandre)!
